summaries/preprocessing/Cleaner.py

```diff
@@ -16,23 +16,23 @@ class Cleaner:
     min_length_reference: int
     min_length_summary: int
     length_metric: str
-    ngram_similarity_range: Optional[Tuple[float, float]]
+    ngram_similarity_range: Optional[Union[Tuple[float], List[float]]]
```

Comment:
See https://stackoverflow.com/questions/40181344/how-to-annotate-types-of-multiple-return-values:

> You really should be returning a tuple, always, if your function produces a fixed number of return values, especially when those values are specific, distinct types.

This implies that the previous way was correct, even if it means slightly less flexible types being passed.